tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -83,92 +140,67 @@ describe(__filename, () => {
   };
 
   it(`calls getClientCompatibility with the add-on's current version if no version is supplied`, () => {
-    const addon = fakeAddon;
-
-    _loadVersions({ slug: addon.slug, versions: [addon.current_version] });
-
     const clientApp = CLIENT_APP_FIREFOX;
-    const _getClientCompatibility = jest.fn().mockReturnValue({
-      compatible: true,
-    });
 
     _dispatchClientMetadata({
       clientApp,
     });
 
-    render({
-      _getClientCompatibility,
-      addon: createInternalAddonWithLang(addon),
-    });
+    renderWithCurrentVersion();
 
     expect(_getClientCompatibility).toHaveBeenCalledWith({
-      addon: createInternalAddonWithLang(addon),
+      addon: createInternalAddonWithLang(fakeAddon),
       clientApp,
-      currentVersion: createInternalVersionWithLang(addon.current_version),
+      currentVersion: createInternalVersionWithLang(fakeAddon.current_version),
       userAgentInfo: store.getState().api.userAgentInfo,
     });
   });
 
   it(`calls getClientCompatibility with a specific version if supplied`, () => {
-    const slug = 'some-slug';
-    const addon = { ...fakeAddon, slug };
     const version = { ...fakeVersion, id: fakeVersion.id + 1 };
 
-    _loadVersions({ slug, versions: [version] });
+    _loadVersions({ slug: fakeAddon.slug, versions: [version] });
```

Comment:
Along the same lines, I'm not crazy about using `fakeAddon` here. That is assuming that `fakeAddon` was also used somewhere else, and I don't think we should be assuming that.  What we could maybe do, which would accomplish something similar, is to have `_loadVersions` have a default value for `slug`, which is the same as the default value for `slug` elsewhere, and then just not pass `slug` at all when we know we want all the defaults used. I have something like this in a few test suites where I define a `defaultSlug` `const`, which is then used as the default value for some helper functions, and I can also use it specifically in tests, knowing that it is the *default* slug. There's just something about using `fakeAddon` and relying on the fact that it's also used elsewhere that doesn't sit right with me.

Reply:
How does 8afe5e52697f0dd9bd4b089f584c6c3399395594 look ?